packages/components/src/navigation/index.tsx

```diff
@@ -76,9 +84,14 @@ export default function Navigation( {
 		<NavigationUI className={ classes }>
 			<div
 				key={ menu }
-				className={ classnames( {
-					[ animateClassName ]: isMounted.current && slideOrigin,
-				} ) }
+				className={
+					animateClassName
+						? classnames( {
+								[ animateClassName ]:
+									isMounted.current && slideOrigin,
+						  } )
+						: undefined
+				}
```

Comment:
⚠️ Runtime change:

Since `animateClassName` can be `undefined`, I added this extra check to avoid using an `undefined` value as an object's key.